README.md

```diff
@@ -40,22 +40,22 @@ This gem requires that you have the following credentials:
 rails new my_shopify_app
 ```
 
-2. Add the Shopify App gem to `my_shopify_app`'s Gemfile.
+2. Add the Shopify App gem to app's Gemfile.
 
 ```sh
 bundle add shopify_app
 ```
 
-3. Create a `.env` file in the root of `my_shopify_app` to specify your full host and Shopify API credentials:
+3. You will need to provide several environment variables to the app.
+There are a variety of way of doing this, but for a development environment we recommended the [`dotenv-rails`](https://github.com/bkeepers/dotenv) gem.
+Create a `.env` file in the root of your Rails app to specify the full host and Shopify API credentials:
```

Comment:
I thought it was misleading to mention using `.env` before the `dotenv-rails` gem is mentioned.

Reply:
@andyw8  env file should not be recommended. It is safer to have `credentials.yml.` It should be listed first imo.

https://edgeguides.rubyonrails.org/security.html#environmental-security